README.md

```diff
@@ -1,7 +1,141 @@
-# Mattermost Boards (Focalboard Plugin)
+> [!WARNING]
+> **Effective September 15th, 2023, Mattermost, Inc. staff are no longer reviewing or merging pull requests for either Focalboard or the Mattermost Boards plugin in this repository (`mattermost/focalboard`). We encourage the community to fork this repository for continued development and contributions.**
+>
+> The reason behind these changes is to focus Mattermost developer resources on improving the platform’s performance and core features to ensure Mattermost continues being resilient, stable, and best-in-breed for critical operations.
+>
+> ️💡 [Learn more](https://forum.mattermost.com/t/upcoming-product-changes-to-boards-and-various-plugins/16669)
+
+# Mattermost boards plugin 
+
+### A self-hosted plugin for managing boards withing a Mattermost installation
+
+Focalboard is an open source, multilingual, self-hosted project management tool that's an alternative to Trello, Notion, and Asana.
 
 **[Mattermost Boards](https://mattermost.com/boards/)** is the Mattermost plugin version of Focalboard that combines project management tools with messaging and collaboration for teams of all sizes. To access and use **Mattermost Boards**, install or upgrade to Mattermost v6.0 or later as a [self-hosted server](https://docs.mattermost.com/guides/deployment.html?utm_source=focalboard&utm_campaign=focalboard) or [Cloud server](https://mattermost.com/get-started/?utm_source=focalboard&utm_campaign=focalboard). After logging into Mattermost, select the menu in the top left corner of Mattermost and select **Boards**.
 
-***Mattermost Boards** is installed and enabled by default in Mattermost v6.0 and later.*
+![Focalboard](website/site/static/img/hero.jpg)
+
+## Try Focalboard
```

Comment:
Is the official name ``Mattermost Boards plugin`` or ``Focalboard plugin``? Please ensure all references are updated accordingly.

Reply:
I'd recommend `Mattermost Boards` or `Mattermost Boards plugin` depending on the context. We should not make any references in this repo to "focalboard" IMO, as it will likely cause more confusion

Reply:
Sure.

Reply:
Sure.